src/blocks/kdeconnect.rs

```diff
@@ -34,22 +37,29 @@
 //! ```toml
 //! [[block]]
 //! block = "kdeconnect"
-//! format = " $icon {$bat_icon $bat_charge|}{ $notif_icon|} "
+//! format = " $icon {$bat_icon $bat_charge |}{$notif_icon |}{$network_icon$network $network_type |}"
 //! bat_good = 101
 //! ```
 //!
 //! # Icons Used
 //! - `bat` (as a progression)
 //! - `bat_charging` (as a progression)
+//! - `net_wireless` (as a progression)
```

Comment:
It might be a good idea to use specialized icons, such as `nf-md-network_strength*`. Font awesome also has some signal icons, but I'm not sure if they're free...

Reply:
I used,
- https://fontawesome.com/icons/signal?f=classic&s=solid on fontawecome6
- https://www.nerdfonts.com/cheat-sheet and `nf-md-network_strength*` from it for material-nf
- https://fonts.google.com/icons?selected=Material+Symbols+Outlined:network_cell:FILL@0;wght@400;GRAD@0;opsz@48&icon.query=network+cell for material theme
- https://fontawesome.com/v4/icon/wifi for awesome4
- https://fontawesome.com/v5/icons/wifi?f=classic&s=solid for awesome5

Turns out I don't have those fonts installed, and when I tried to install some and test I only got unicode boxes, so I can't verify other ones work. The awesome6 one work as I've verified it.
Please check my latest commit and verify those if you can.

Reply:
`net_strength` can be `\uf012` for all of the font-awesome versions. It was introduced in version 1.0.0. https://fontawesome.com/v4/icon/signal.
material appears to have a progression available: https://fonts.google.com/icons?selected=Material+Symbols+Outlined:signal_cellular_0_bar:FILL@0;wght@400;GRAD@0;opsz@48&icon.query=signal+cellular
For material-nf you put 6 elements in the progression, but there are only 5 states (0..=4). I'd drop `nf-md-network_strength_off_outline`

Reply:
Ohh, I somehow didn't find it by searching "network" "cell" like in v6, so I thought it wasn't there. I'll change that.

I'll add the material one too.

Oh! In my mind I was thinking of 5 states `0..=4` and then -1 as unavailable, but I guess we make that 0 to make it into progression. Might be better to add 1 to the value obtained and then turn it into `0..=5`. SIM disabled (-1) and no/very weak signal (0), might warrant a difference.